modules/exploits/multi/php/wordpress_duplicator.rb

```diff
@@ -0,0 +1,117 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ManualRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'            => 'Snapcreek Duplicator WordPress plugin installer.php code injection',
+      'Description'     => %q{
```

Comment:
Your spacing is a little long on the description, try to keep it at 75 or 80 characters (i'm sure there is an official number somewhere)